frontend/src/features/comparisons/ComparisonList.tsx

```diff
@@ -1,22 +1,15 @@
 import React from 'react';
 
-import {
-  Grid,
-  Container,
-  makeStyles,
-  Theme,
-  Tooltip,
-  Fab,
-  Box,
-} from '@material-ui/core';
-import { Compare as CompareIcon } from '@material-ui/icons';
+import { Grid, Container, Theme, Tooltip, Fab, Box } from '@mui/material';
+import makeStyles from '@mui/styles/makeStyles';
+import { Compare as CompareIcon } from '@mui/icons-material';
 import type { Comparison } from 'src/services/openapi';
 import { VideoCardFromId } from '../videos/VideoCard';
 
 const useStyles = makeStyles((theme: Theme) => ({
   content: {
     padding: theme.spacing(3),
-    [theme.breakpoints.down('xs')]: {
+    [theme.breakpoints.down('sm')]: {
```

Comment:
I personally prefer the previous look on mobile phone (with "xs") :
![Capture d’écran de 2021-12-30 06-42-02](https://user-images.githubusercontent.com/47578089/147724963-77a66263-24fb-442b-99b0-e2fce8b8c788.png)
 compared to the new look (with "sm"), even if it leaves less unused space :
![Capture d’écran de 2021-12-30 06-40-00](https://user-images.githubusercontent.com/47578089/147725077-669920f3-4b16-44a5-9989-f9b93bb68d3f.png)